lib/query_builder/cql/operators/cql_gt.rb

```diff
@@ -11,13 +11,21 @@ module QueryBuilder::CQL::Operators
   #   fn[:foo]
   #   # => "foo > 3"
   #
+  # It also support nested CQL function calls
+  #
+  # @example
+  #   cql_fn = Operators[:cql_mintimeuuid, "2021-03-25 00:00:00 +0000"]
+  #   fn = Operators[:cql_gt, cql_fn]
+  #   fn[:foo]
+  #   # => "foo > MINTIMEUUID('2021-03-25 00:00:00')"
+  #
   # @param [#to_s] column
-  # @param [Numeric] value
+  # @param [Transproc::Function || Numeric] argument
```

Comment:
I'm not that well-versed in YARD, but it looks like multiple types are separated with a `,` (comma): https://rubydoc.info/gems/yard/file/docs/Tags.md#param

Maybe?
```ruby
@param [Transproc::Function, Numeric]
```

Reply:
Done!